src/arp.cpp

```diff
@@ -76,13 +76,13 @@ void arpReset() {
 void arpTick() {
 	if (voice_state.n_held_keys() > 0) {
 		if (preset_data.arp_mode) {
-			if (preset_data.lfo[0].retrig) {
+			if ((preset_data.lfo[0].retrig) || (voice_state.n_held_keys() == 1)) { // always retrigger on the first key
```

Comment:
You don't need that many braces.
```suggestion
			if (preset_data.lfo[0].retrig || voice_state.n_held_keys() == 1) { // always retrigger on the first key
```

The `==` operator binds stronger than the `&&` or `||` operators (where `&&` is still stronger than `||`). This is very similar to school math, where `*` binds stronger than `+`.